backend/internal/domain/helpers/password_hasher.go

```diff
@@ -0,0 +1,26 @@
+package helpers
+
+import (
+	"errors"
+
+	"golang.org/x/crypto/bcrypt"
+)
+
+const (
+	DefaultCost int = 10
+)
+
+func HashPassword(password string) (string, error) {
+	if len(password) > 36 {
+		ErrPasswordTooLong := errors.New("password length should not exceed 72 bytes")
```

Comment:
variable names should always be lowercase
Question: how do you know that the len(password) == 36 is exactly 72 bytes?
It is utf8 encoded, it can be 36bytes up to 4*36

Reply:
Read `len` documentation. Looks like it returns bytes numbers, not string length. Fixed

Reply:
You can always leverage `utf8.RuneCountInString` to count number of characters :relaxed:

Reply:
Thank you :)